Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -239,10 +239,13 @@ - (WMFContentGroup *)sectionForIndexPath:(NSIndexPath *)indexPath {
         content = [content bk_map:^id(WMFFeedTopReadArticlePreview *obj) {
             return [obj articleURL];
         }];
+    } else if ([group contentType] == WMFContentTypeStory) {
+        content = [content wmf_mapAndRejectNil:^id(WMFFeedNewsStory *obj) {
```

Comment:
rejecting nil may not be safe because we don't do this same logic when reporting the count of cells in the section.